client/README.md

```diff
@@ -11,8 +11,13 @@ This extension provides a Yseop Markup Language (YML) basic support for VS Code.
     * `granule`: Text granule
     * `instance`: Object instance
     * `textu`: Textualization attribute
+![Snippet for textualization Gif.](images/textualization.gif "Snippet for textualization.")
+
 - Code folding
 - KB class attributes completion (needs class to be opened at least once)
+- Basic Yseop Engine model based type completion
+    * This feature uses the parameter `yseopml.pathToPredefinedObjectsXml` to get the **absolute** path of the `predefinedObjects.xml` file provided by user's version of Yseop Engine.
```

Comment:
* No need to put so much stress on “absolute”. This will scare away the few ones that might not know what that means. Someone that wants to use a relative one will soon realize he has no idea what the starting directory should be, so…
* Path _to_ a file. “Of” makes it sound like it's an unique property or something.
* “user's version of Yseop Engine.” → grammatically weird. Plus, I guess the user can have several engine versions. Suggestion on the fly: “provided with one of the user's Yseop Engine versions”?

Reply:
Ok, I'll do the changes.